### ISSUE TITLE
Current auth endpoints will be deprecated on December 1, 2021

### ISSUE THREAD

**Issue by smonn** [homebridge-simplisafe3/homebridge-simplisafe3#231]
### Describe The Bug:

The authentication endpoints `/api/token` and `/api/mfa/challenge` respond with a field stating:

> This authentication endpoint is deprecated and may no longer function after December 1, 2021.

Afraid I don't know what the new authentication endpoints might be.

### To Reproduce:

Check the JSON response when calling one of the login and/or MFA challenge endpoints.

### Expected behavior:

Upgrade to the non-deprecated endpoints, whatever those might be.

### Environment:

* **Plugin version**: N/A
* **Node.js Version**: N/A
* **NPM Version**: N/A
* **Homebridge Version**: N/A
* **Operating System**: N/A

### Homebridge Config:

N/A

### Screenshots:

N/A

### Logs:

N/A

**Comment by shamoon** [comment]
Good find, thanks for pointing this out.

As I guessed, looks like theyre going to release "v2" of the API, i.e. new endpoint is `https://api.simplisafe.com/v2/api/token` but something tells me its not going to be as simple as changing the URL. Going to look into this, good news is we have plenty of time...

**Comment by shamoon** [comment]
Good find, thanks for pointing this out. Going to look into this, good news is we have plenty of time...

**Comment by shamoon** [comment]
Bad bot

(Solution is in progress...)

**Comment by shamoon** [comment]
@nzapponi Hope all's well man! I was thinking about trying to release this as a beta (see oauth branch) but Im not sure I have npm access? NBD if its complicated but might be helpful if poss...

**Comment by sterumbelow** [comment]
Hi, Michael. 

I've updated to the latest version and have run into a string of errors. Accessories are now gone from Homebridge and of course my home app too. 

My login details are correct and I've changed no other settings. 

Anything I can do to fix this?
Thanks,
Ste. 

![image](https://user-images.githubusercontent.com/68124742/135282270-7e6d9e8f-76da-41df-9b92-b6ef2a226a31.jpeg)

![image](https://user-images.githubusercontent.com/68124742/135282289-a2193333-e0c9-464e-80d0-661f19c22631.jpeg)
